docs/tutorials/github.md

```diff
@@ -104,6 +130,6 @@ Now it's time to run our first trestlebot action!  We will go ahead and create o
 
 6. Click `Run Workflow`
 
-Once the workflow has completed you should have a new Pull Request containing the files trestlebot generated for the component definition.  After reviewing the files you can go ahead and merge the PR!
+Once the workflow job has completed, there will be a new Pull Request containing the files trestlebot generated for the component definition.  After reviewing the commit changes, the Pull Request can then be merged into the main branch!
```

Comment:
"commit changes" --> "committed changes"?

Reply:
Hi @gvauter, sorry for delay. Those minor changes are now committed in the merge request. Please let me know if you see any other fixes needed prior to giving it your "go ahead."

Reply:
Looks good to me, thanks @benroose!